packages/backend/src/managers/applicationManager.ts

```diff
@@ -345,13 +346,22 @@ export class ApplicationManager {
     }
 
     // create new pod
+    const labels = {
+      [LABEL_RECIPE_ID]: recipe.id,
+      [LABEL_MODEL_ID]: model.id,
+    };
+    const modelPorts = images.filter(img => img.modelService).flatMap(img => img.ports);
```

Comment:
Any reason why you also add the model port? As we do not ping the model anymore to see if it's running we should not expose it in the pod, no?

Reply:
In the design made by Mo (https://github.com/projectatomic/ai-studio/issues/235), the port of the model is displayed. I can imagine the developer may want to know the model's port, to make some manual requests on it, for example

Reply:
maybe let's ask @jeffmaury . BTW looks good to me for the rest 👍